### ISSUE TITLE
Enquiry about the results for tsp-gls

### ISSUE THREAD

**Issue by SnoopD201** [ai4co/reevo#16]
Hi, I have a question regarding Table 1 in the paper. For the tsp-gls results declared in the table, does the EoH result follow the original implementation described in the EoH paper, or is it based on your implementation in ReEvo? I noticed that the signature of the function for optimization for tsp-gls differs between EoH and ReEvo. Could you clarify which implementation was used to get your EoH results in Table 1?
Thank you!

**Comment by henry-yeh** [comment]
Hi there, thanks for your interest.

We use the exact implementation of the original EoH repo.

**Comment by SnoopD201** [comment]
OK, got it. But there is another question regarding the optimal gap for EoH presented in Table 1, during my test, it seems that this best result cannot be consistently achieved in every run, most of the time the best solution obtained by EoH on tsp-gls is worse than the reported result. Is this a normal phenomenon when you conducted the experiments? It feels like the initialization of the seed algorithm plays a crucial role in the whole optimization process.
Thanks!

**Comment by henry-yeh** [comment]
We haven't conducted EoH training/evolution on TSP_GLS. But, indeed, LHHs often show high variance across different runs and are sensitive to hyperparameters including the seed algorithm.

**Comment by SnoopD201** [comment]
Understood. Thanks for your explanation.
